go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -361,9 +352,16 @@ func getIPAsHexString(ip net.IP) string {
 
 // EnsureHybridOverlayBridge sets up the hybrid overlay bridge
 func (n *NodeController) EnsureHybridOverlayBridge(node *kapi.Node) error {
-	if n.initialized {
+	if n.initialized && len(n.gwLRPIP) > 0 {
```

Comment:
@JacobTanenbaum I don't think we need this check? Any time initialized = true we'll have already successfully parsed gwLRPIP right?